src/const.js

```diff
@@ -243,43 +244,67 @@ export const navLink = [
 export const data = [
     {
         id: 1,
-        image: biakitchen,
-        title: "Bia's Baking Kitchen",
-        github: "https://github.com/Abdullah-Logic/bias-baking-kitchen.git",
+        image: edusity,
+        title: "Edusity Website",
+        desc: "This is a test clone of the Edusity website replicating its features and functionality as a practice project,",
```

Comment:
No need to mention "test" and "as a practice project" . You can mention at the end of description "(Practice Project)" text should be light, small  (Check all cards.)